go.mod

```diff
@@ -11,3 +11,5 @@ require (
 	google.golang.org/grpc v1.40.0
 	k8s.io/klog/v2 v2.60.0
 )
+
+replace golang.org/x/text => golang.org/x/text v0.3.7
```

Comment:
I am not seeing any code change. Do we need to run "go mod vendor"?

Reply:
I have run that `go mod vendor`, there is really no code change, while after image build, `trivy` tells me that cve is gone.

Reply:
```
# docker build -t andyzhangx/livenessprobe:latest2 -f Dockerfile --output=type=docker --label revision=v2.7.0-0-g5daa5c2 .
# trivy --ignore-unfixed andyzhangx/livenessprobe:latest2
2022-05-14T02:54:59.990Z        INFO    Need to update DB
2022-05-14T02:54:59.990Z        INFO    Downloading DB...
27.85 MiB / 27.85 MiB [------------------------------------------------------------------------------------------------------------] 100.00% 13.88 MiB p/s 2s
2022-05-14T02:55:02.815Z        INFO    Detected OS: debian
2022-05-14T02:55:02.815Z        INFO    Detecting Debian vulnerabilities...
2022-05-14T02:55:02.815Z        INFO    Number of language-specific files: 1
2022-05-14T02:55:02.815Z        INFO    Detecting gobinary vulnerabilities...

andyzhangx/livenessprobe:latest2 (debian 11.3)
==============================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)


livenessprobe (gobinary)
========================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

Reply:
I guess the vulnerability is in a package we don't use